pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>spring-boot-starter-parent</artifactId>
         <groupId>org.springframework.boot</groupId>
-        <version>2.7.12</version>
+        <version>3.0.0</version>
```

Comment:
There is latest versions in 3.x.x. might have resolved many vulnerabilities. Try to uplift to the latest.

Reply:
done